### PR TITLE
YLP-497 Add HCP private pratice filter

### PR DESCRIPTION
**Pull request by yvon-dblg**

### CODE REVIEW

File: packages/yourloops/pages/hcp/patients-list.tsx [hunk 1]
```diff
@@ -256,6 +255,9 @@ class PatientListPage extends React.Component<RouteComponentProps, PatientListPa
         case "pending":
           return false; // TODO
         default:
+          if ((patient.teams?.includes(filterType) ?? false) === false) {
```
Comment (leovalette) [inline packages/yourloops/pages/hcp/patients-list.tsx:258]:
```suggestion
          if ((!patient.teams?.includes(filterType)) {
```

File: packages/yourloops/pages/hcp/patients-list.tsx [hunk 2]
```diff
@@ -274,7 +276,7 @@ class PatientListPage extends React.Component<RouteComponentProps, PatientListPa
     } else if (filterType === "pending") {
       patients = []; // TODO
     } else if (filterType !== "all") {
-      // TODO
+      patients = allPatients.filter((patient: User): boolean => patient.teams?.includes(filterType) ?? false);
```
Comment (leovalette) [inline packages/yourloops/pages/hcp/patients-list.tsx:279]:
```suggestion
      patients = allPatients.filter((patient: User): boolean => patient.teams?.includes(filterType));
```

**Review by leovalette — APPROVED** [review]

File: packages/yourloops/pages/hcp/patients-list.tsx (hunk 1 above)
Comment (yvon-dblg) [reply packages/yourloops/pages/hcp/patients-list.tsx:258]: My next PR will change this code

File: packages/yourloops/pages/hcp/patients-list.tsx (hunk 2 above)
Comment (yvon-dblg) [reply packages/yourloops/pages/hcp/patients-list.tsx:279]: My next PR will change this code

File: packages/yourloops/pages/hcp/patients-list.tsx (hunk 1 above)
Comment (yvon-dblg) [reply packages/yourloops/pages/hcp/patients-list.tsx:258]: Fix in https://github.com/mdblp/blip/pull/400

File: packages/yourloops/pages/hcp/patients-list.tsx (hunk 2 above)
Comment (yvon-dblg) [reply packages/yourloops/pages/hcp/patients-list.tsx:279]: Fix in https://github.com/mdblp/blip/pull/400

**Review by franck-fourel — APPROVED** [review]
LGTM

**Pull request merged**